code/modules/projectiles/projectile.dm

```diff
@@ -43,32 +43,43 @@
 	///greyscale support
 	greyscale_config = null
 	greyscale_colors = null
-
 	///Any special effects applied to this projectile
 	var/flags_projectile_behavior = NONE
-
+	///Hit impact sound
 	var/hitsound = null
-	var/datum/ammo/ammo //The ammo data which holds most of the actual info.
-
-	var/def_zone = BODY_ZONE_CHEST	//So we're not getting empty strings.
-
+	///The ammo data which holds most of the actual info
+	var/datum/ammo/ammo
+	///target def zone
+	var/def_zone = BODY_ZONE_CHEST
+	///the pixel X location of the tile that the player clicked. Default is the center
 	var/p_x = 16
-	var/p_y = 16 // the pixel location of the tile that the player clicked. Default is the center
-	var/apx //Pixel location in absolute coordinates. This is (((x - 1) * 32) + 16 + pixel_x)
-	var/apy //These values are floats, not integers. They need to be converted through CEILING or such when translated to relative pixel coordinates.
-
-	var/atom/shot_from 	 = null // the object which shot us
-	var/turf/starting_turf = null // the projectile's starting turf
-	var/atom/original_target = null // the original target clicked
-	var/turf/original_target_turf = null // the original target's starting turf
-	var/atom/firer 		 = null // Who shot it
-
-	var/list/atom/movable/uncross_scheduled = list() // List of border movable atoms to check for when exiting a turf.
-
+	///the pixel y location of the tile that the player clicked. Default is the center
+	var/p_y = 16
+	//Pixel location in absolute coordinates. This is (((x - 1) * 32) + 16 + pixel_x)
+	//These values are floats, not integers. They need to be converted through CEILING or such when translated to relative pixel coordinates.
+	///Projectile X coordinate
```

Comment:
May want to use 
/**
 *
 */

Reply:
top 2 lines aren't autodoc, they relate to both vars. I didn't really want to copy paste all the text for both